fix(slides): expose interface to provide custom animations (#17959)
ionic-team/ionic: core/src/components/slides/readme.md

Licensed under MIT

### Custom Animations

> By default, Ionic ships with the `slide` animation effect built in. Custom animations can be provided via the `options` prop. Examples for other animations can be found below.

Reviewer: ```suggestion
By default, Ionic slides use the built-in `slide` animation effect. Custom animations can be provided via the `options` property. Examples of other animations can be found below.
```
